v2.0/resourcepacks/create_astral_translation/assets/doodads/lang/zh_cn.json

```
@@ -1,3 +1,117 @@
 {
-    "block.doodads.asphalt": "安山岩路面"
-}
\ No newline at end of file
+    "itemGroup.doodads.general": "Doodads",
+    "item.doodads.pebble": "鹅卵石",
+    "item.doodads.slingshot": "弹弓",
+    "item.doodads.beaver_teeth": "海狸牙",
+    "item.doodads.beaver_teeth.tooltip.shift": "瞬间破坏木头或者给木头剥皮",
+    "item.doodads.beaver_teeth.tooltip2.shift": "呃，就像海狸一样",
+    "item.doodads.ender_goggles": "末影护目镜",
+    "item.doodads.ender_goggles.tooltip.shift": "可以遮蔽末影人对你视线的感知",
+    "item.doodads.speed_boots": "疾跑鞋",
+    "item.doodads.speed_boots.tooltip.shift": "⻊包 白勺 飞 忄夬",
+    "item.doodads.bendy_straw": "弯曲吸管",
+    "item.doodads.bendy_straw.tooltip.shift": "使药剂更持久",
+    "item.doodads.magic_plum": "魔法梅子",
+    "item.doodads.magic_plum.tooltip.shift": "一种消耗型图腾",
+    "item.doodads.paintbrush": "漆刷",
+    "item.doodads.paintbrush.tooltip.shift": "主手拿染料，副手拿刷子",
+    "item.doodads.paintbrush.tooltip2.shift": "拿着水桶会清洗砌块",
+    "item.doodads.paintbrush.tooltip3.shift": "如果没有染料，它将选择随机颜色",
+    "item.doodads.scissors": "剪刀",
+    "item.doodads.scissors.tooltip": "不要拿剪刀当玩具!",
+    "item.doodads.duct_tape": "强力胶带",
+    "item.doodads.duct_tape.tooltip.shift": "对怪物使用可以让他们安静下来[可以用剪刀剪掉]",
+    "item.doodads.soggy_glove": "潮湿手套",
+    "item.doodads.soggy_glove.tooltip.shift": "可以让混凝土粉末立刻变成混凝土",
+    "item.doodads.loggers_glove": "伐木工手套",
+    "item.doodads.loggers_glove.tooltip.shift": "避免在使用斧头的时候给原木剥皮",
+    "item.doodads.sun_ring": "日耀之戒",
+    "item.doodads.sun_ring.tooltip.shift": "在白天的时候给予伤害提升buff",
+    "item.doodads.moon_ring": "月辉之戒",
+    "item.doodads.moon_ring.tooltip.shift": "在晚上的时候给予伤害提升buff",
+    "item.doodads.celestial_ring": "辉耀之戒",
+    "item.doodads.celestial_ring.tooltip.shift": "全天候都给予伤害提升buff",
+    "item.doodads.cactus_ring": "仙人掌之戒",
+    "item.doodads.cactus_ring.tooltip": "给攻击者造成伤害",
+    "item.doodads.midnights_eye": "夜影之眼",
+    "item.doodads.midnights_eye.tooltip": "提供夜视效果",
+    "item.doodads.slimey_shoes": "史莱姆靴",
+    "item.doodads.slimey_shoes.tooltip": "给予跳跃提升并免疫掉落伤害",
+    "item.doodads.glare_plushie": "光耀绒毛",
+    "item.doodads.glare_plushie.tooltip": "在暗处自动召唤一个漂浮光源",
+    "item.doodads.glare_staff": "光耀法杖",
+    "item.doodads.glare_staff.tooltip": "召唤一个漂浮光源",
+    "item.doodads.shulker_aglet": "潜影贝箍",
+    "item.doodads.shulker_aglet.tooltip": "免疫飘浮效果",
+    "item.doodads.blossom_belt": "百花带",
+    "item.doodads.blossom_belt.tooltip": "免疫中毒效果",
+    "item.doodads.generic_tooltip": "按下§eSHIFT§r获取更多信息",
+    "item.doodads.rubber_band": "橡皮筋",
+    "item.doodads.spore_sword": "孢子剑",
+    "item.doodads.spore_sword.tooltip": "给目标中毒效果",
+    "item.doodads.doodad_bundle": "Doodad 盲盒",
+    "item.doodads.doodad_bundle.tooltip": "内含一个随机的 Doodad 物件",
+    "block.doodads.glare_light": "日光灯",
+    "block.doodads.platform": "木制平台",
+    "block.doodads.asphalt": "沥青",
```

```suggestion
    "block.doodads.asphalt": "安山岩路面",
```
整合包把沥青改为了安山岩路面（Andesite Pavement）。